index.html

```diff
@@ -211,23 +210,25 @@
 
         <div class="divisoria"></div>
 
-        <div class="novidades">árvores da vida</div>
+        <a href="arvore da vida.html" class="novidades">árvores da vida</a>
```

Comment:
É melhor usar `snake case` no nome dos arquivos
```suggestion
        <a href="arvore_da_vida.html" class="novidades">árvores da vida</a>
```

Reply:
Alterado